modules/oadp-review-backup-restore.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/backing_up_and_restoring/restoring-applications.adoc
+// * backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-review-backup-restore_{context}"]
+= Previewing resources before running backup and restore
+
+{oadp-short} backs up application resources by type, namespace, or label at runtime. This means, you can view the resources after the backup is complete. Similarly, you can view the restored objects by namespace, PV, or label after a restore operation is complete. To preview the resources in advance, you can do a dry run of the backup and restore operations. 
+
+.Prerequisites
+
+* You must install the OADP Operator.
+
+.Procedure
+
+. To preview the resources that will be backed up before running the actual backup, run the following `velero` command:
++
+[source,terminal]
+----
+$ velero backup create <backup-name> --snapshot-volumes false # <1>
+----
+<1> Specify `--snapshot-volumes false`.
++
+. To know more details about the backup resources, run the following `velero` command:
++
+[source,terminal]
+----
+$ velero describe backup <backup_name> --details # <1>
+----
+<1> Specify the name of the backup.
++
+. To preview the resources that will be restored before running the actual restore, run the following `velero` command:
++
+[source,terminal]
+----
+$ velero restore create --from-backup <backup-name> # <1>
+----
+<1> Specify the name of the backup created to review the backup resources.
++
+[IMPORTANT]
+====
+The `velero restore create` command creates restore resources in the cluster. Make sure to delete the resources created by the restore, after reviewing the resources.
+====
++
+. To know more details about the restore resources, run the following `velero` command:.
++
+[source,terminal]
+----
+$ velero describe restore <restore_name> --details # <1>
```

Comment:
```suggestion
$ velero describe restore <restore_name> --details <1>
```

Reply:
Hi @anarnold97 Thanks for pitching in. I agree to your point that you need to comment out the callout in a YAML file as per the doc guidelines. But, since this is a codeblock we do not need to comment out the code blocks.
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#callouts-in-code-blocks

Reply:
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#using-asciidoc-callouts-in-the-yaml